### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by Twitch-Antho** [Herysia/AmongUsUnknownImpostors#1]
Good morning,
Your mod is amazing and my Discord team like to make it popular since it's getting more play since it's not updated, can you update it?
I ask you very kindly and it would make us very happy

I am French = text created with google translate

**Comment by Herysia** [comment]
Hey i'm sorry i'm not into Among us anymore. If you wish to give it a try at updating, it has been made using Reactor (https://reactor.gg/discord). Which is still being updated, so hopefully that part of the game hasn't changed that much and you can just update the dependecies (reactor). And maybe get rid of lobbyOptionsAPI

**Comment by Twitch-Antho** [comment]
Yes, but so in your mod I have to update Reactor and lobbyOptionsAPI? and how do I update a file?
But the problem is that there have been bigs up to date with the game.

Je suis français = texte créé avec google translation

**Comment by Herysia** [comment]
The parts modified by this mod are unlikely to have changed much in the game. So hopefully if you manage to update reactor it'll work.
You can follow the reactor documentation (get started) to install all dependencies:
https://docs.reactor.gg/
Then you can edit the [AmongUsUnknownImpostors.csproj](https://github.com/Herysia/AmongUsUnknownImpostors/blob/master/AmongUsUnknownImpostors/AmongUsUnknownImpostors.csproj) file with the correct versions.

Regarding optionsAPI, it is not mandatory, so you can either update it as well, or delete everything related to it from this mod

**Comment by Twitch-Antho** [comment]
Thank you, I'll try it myself but maybe I won't be able to do it, it's a shame you don't have the time... I'll check it out over the weekend. If I succeed I will send you the .zip file so that you can upload it to Github.
Just the Among Us versions of the game?

Je suis français = texte créé avec google traduction

**Comment by amonguscn** [comment]
@AnthoYt https://github.com/FangkuaiYa/AmongUs-HomemadePlugin/releases/tag/v2.0.0-AmongUsUnknownImpostors
You can give this a try

**Comment by Twitch-Antho** [comment]
Vous l'avez mis à jour ???
